.docker/entrypoint/lib/entrypoint.rb

```diff
@@ -30,8 +30,9 @@ def initialize_context(context)
         jekyll_data_dir = ENV.fetch('JEKYLL_DATA_DIR', Dir.pwd)
         jekyll_var_dir = ENV.fetch('JEKYLL_VAR_DIR')
         debug = ENV.fetch('DEBUG', false)
+        profile = ENV.fetch('PROFILE', false)
 
-        Context.new(env, jekyll_var_dir, jekyll_data_dir, auth_token, debug)
+        Context.new(env, jekyll_var_dir, jekyll_data_dir, auth_token, debug, profile)
```

Comment:
Codacy found an issue: [Line is too long. [85/80]](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6123667)